toplevel/ccompile.ml

```diff
@@ -119,7 +120,9 @@ let compile opts ~echo ~f_in ~f_out =
       Dumpglob.start_dump_glob ~vfile:long_f_dot_v ~vofile:long_f_dot_vo;
       Dumpglob.dump_string ("F" ^ Names.DirPath.to_string ldir ^ "\n");
       let wall_clock1 = Unix.gettimeofday () in
-      let state = Vernac.load_vernac ~echo ~check:true ~interactive:false ~state long_f_dot_v in
+      let open Stm.AsyncOpts in
+      let state = Vernac.load_vernac ~echo
+          ~check:(stm_options.async_proofs_mode = APoff) ~interactive:false ~state long_f_dot_v in
```

Comment:
This is duplicated with the above and makes lines very long; please factor the check parameter to a `let check = ....` instead.

Reply:
The other change is a mistake. I'll remove it, so no duplication. Will define `check` for this one, thanks!

Reply:
Fixed, thanks!